File: camera_seek_bar/app/src/main/java/com/omkar/camera_seek_bar/MainActivity.java
package com.omkar.camera_seek_bar;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button btnCamera, btnSeekbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnCamera = findViewById(R.id.btnCamera);
        btnSeekbar = findViewById(R.id.btnSeekbar);

        btnCamera.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, cameraActivity.class);
            startActivity(intent);
        });

        btnSeekbar.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, SeekBarActivity.class);
            startActivity(intent);
        });
    }
}

File: doc_viewer/app/src/main/java/com/omkar/docviewer/MainActivity.kt
package com.omkar.docviewer

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.widget.Button
import androidx.activity.result.contract.ActivityResultContracts
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private val picker =
        registerForActivityResult(ActivityResultContracts.OpenDocument()) { uri: Uri? ->
            uri?.let {
                startActivity(
                    Intent(this, ViewerActivity::class.java)
                        .putExtra("uri", it.toString())
                )
            }
        }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<Button>(R.id.openFileBtn).setOnClickListener {
            picker.launch(arrayOf("*/*"))
        }
    }
}
